syncer/online-ddl-tools/online_ddl.go

```diff
@@ -473,7 +473,8 @@ func (r *RealOnlinePlugin) Apply(tctx *tcontext.Context, tables []*filter.Table,
 				if ghostInfo != nil {
 					return ghostInfo.DDLs, nil
 				}
-				return nil, terror.ErrSyncerUnitGhostOnlineDDLOnGhostTbl.Generate(schema, table)
+				return []string{""}, nil
```

Comment:
```suggestion
				return nil, nil
```
Can we avoid this kind of empty string